tests/Glpi/Inventory/tests/units/Converter.php

```diff
@@ -371,7 +372,8 @@ protected function batteryVoltsToConvertProvider()
             ['4365 mV', 4365],
             ['8.2 mV', false],
             ['8 2 V', false],
-            ['8,2 V', false]
+            ['8,2 V', false],
+            ['4.255 V', 4255]
```

Comment:
You altered regex to permit lack of space between number and unit.
```suggestion
            ['4.255V', 4255]
```